src/pages/TransactionsList.vue

```diff
@@ -84,26 +128,40 @@ const { width } = useWindowSize();
 
 const hashType = computed(() => (width.value < HASH_BREAKPOINT ? 'short' : 'full'));
 
-const table = useTable(http.fetchTransactions, { reversed: true });
 const { handleUnknownError } = useErrorHandlers();
 
-onMounted(async () => {
+const transactionsTable = useTable(http.fetchTransactions, { reversed: true });
+const instructionsTable = useTable(http.fetchInstructions);
+
+const listState = computed(() => ({
+  status: status.value,
+  kind: transactionType.value,
+}));
+
+const transactionType = ref<ftm.TabDefaultScreen>('All');
+
+const shouldUseTransactions = computed(() => transactionType.value === 'All');
+
+async function fetchTransactions() {
   try {
-    await table.fetch();
+    if (shouldUseTransactions.value) await transactionsTable.fetch(reactiveOmit(listState.value, 'kind'));
+    else
+      await instructionsTable.fetch({
+        ...reactiveOmit(listState.value, 'status'),
+        transaction_status: listState.value.status,
+      });
```

Comment:
It is not a proper use of `reactiveOmit`. It creates a new `reactive()` object on each function call. Reactive objects should only be created within reactive contexts (e.g. during component setup).

You might want to use a plain `omit` here (might be available at `@vueuse/shared`?)